internal/controller/driver_controller.go

```diff
@@ -921,6 +923,22 @@ func (r *driverReconcile) reconcileControllerPluginDeployment() error {
 	return err
 }
 
+func (r *driverReconcile) ControllerPluginCsiAddonsContainerPort() corev1.ContainerPort {
+
+	var port corev1.ContainerPort
+
+	// the cephFS and rbd drivers need to use different ports
+	// to avoid port collisions with host network.
+	port = utils.ControllerPluginCsiAddonsContainerRbdPort
```

Comment:
```suggestion
	// the cephFS and rbd drivers need to use different ports
	// to avoid port collisions with host network.
	port := utils.ControllerPluginCsiAddonsContainerRbdPort
```